addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -63,21 +63,23 @@ def _compute_department_id(self):
     # ORM overrides
     # ----------------------------------------------------
 
-    @api.model
-    def create(self, values):
-        # compute employee only for timesheet lines, makes no sense for other lines
-        if not values.get('employee_id') and values.get('project_id'):
-            if values.get('user_id'):
-                ts_user_id = values['user_id']
-            else:
-                ts_user_id = self._default_user()
-            values['employee_id'] = self.env['hr.employee'].search([('user_id', '=', ts_user_id)], limit=1).id
-
-        values = self._timesheet_preprocess(values)
-        result = super(AccountAnalyticLine, self).create(values)
-        if result.project_id:  # applied only for timesheet
-            result._timesheet_postprocess(values)
-        return result
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            # compute employee only for timesheet lines, makes no sense for other lines
+            if not vals.get('employee_id') and vals.get('project_id'):
+                if vals.get('user_id'):
+                    ts_user_id = vals['user_id']
+                else:
+                    ts_user_id = self._default_user()
+                vals['employee_id'] = self.env['hr.employee'].search([('user_id', '=', ts_user_id)], limit=1).id
```

Comment:
Isn't there a way to extract this search outside of the loop ? Or maybe use the `employee_id` field on the user ?

Reply:
Done :+1: